test/optimization/test_qaoa.py

```diff
@@ -156,8 +157,8 @@ def cb_callback(eval_count, parameters, mean, std):
         x = sample_most_likely(result.eigenstate)
         graph_solution = max_cut.get_graph_solution(x)
 
-        if init_pt is None:       # If None the preferred initial point of QAOA variational form
-            init_pt = [0.0, 0.0]  # i.e. 0,0 should come through as the first point
+        if init_pt is None:  # If None the preferred initial point of QAOA variational form
+            init_pt = first_pt
```

Comment:
The test with None as the initial point is now really meaningless. This was to check that QAOA was correctly using the initial by checking the params of the first callback matched what was expected. About the only thing I might suggest would be to seed aqua_globals and then assert the init point was not 0,0 (or is the random value that we expect which should be fixed when seeded).

Reply:
I added seed and test against the expected values now.